src/main/java/lotto/domain/LottoNumbers.java

```diff
@@ -1,56 +1,31 @@
 package lotto.domain;
 
-import lotto.constant.LottoRanking;
-
-import java.util.ArrayList;
 import java.util.List;
 
 public class LottoNumbers {
     private static final int LOTTO_NUMBER = 6;
 
     private final List<LottoNumber> lottoNumbers;
 
-    private LottoNumber bonusNumber;
-
     public LottoNumbers(List<LottoNumber> lottoNumbers) {
         if (lottoNumbers.size() != LOTTO_NUMBER) {
             throw new IllegalArgumentException(LOTTO_NUMBER + "개의 로또 번호를 입력해 주세요.");
         }
         this.lottoNumbers = lottoNumbers;
     }
 
-    public LottoNumbers(List<LottoNumber> lottoNumbers, LottoNumber bonusNumber) {
-        this(lottoNumbers);
-        if(lottoNumbers.contains(bonusNumber)){
-            throw new IllegalArgumentException(bonusNumber + "는 이미 당첨된 번호입니다.");
-        }
-        this.bonusNumber = bonusNumber;
-
-    }
-
     public List<LottoNumber> getLottoNumbers() {
         return lottoNumbers;
```

Comment:
```suggestion
        return Collections.unmodifiableList(lottoNumbers);
```
외부에서 변경을 막기위해 이렇게도 사용합니다 :)